docker-entrypoint.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/sh
+
+# Copyright 2016, EMC, Inc.
+
+# set -x
+mkdir -p /RackHD/web-ui
+cp -a -f /RackHD/web-ui-base/* /RackHD/web-ui/
+
+while true; do
+  date;
+  sleep 120; # 2 minutes
```

Comment:
sleep in while true so docker never exit.
this is just a debug info: every 120s print date to show this docker container is alive.